src/main/java/com/azure/cosmos/sample/ReadManyItems.java

```diff
@@ -76,178 +79,165 @@ public static void main(String[] args) {
             p.readManyItemsDemo();
             System.out.println("Demo complete, please hold while resources are released");
         } catch (Exception e) {
-            System.out.println("Cosmos getStarted failed with: "+ e);
+            System.out.println("Cosmos getStarted failed with: " + e);
```

Comment:
Instead of using system.out.println, it is important to show customers we support log4j APIs. Please use log4j for all logging purpose. 
Errors can be logged using `logger.error()`, and other information can be logged using `logger.info()` APIs. 
Here is an example on how to do it - https://github.com/Azure-Samples/azure-cosmos-java-sql-api-samples/blob/main/src/main/java/com/azure/cosmos/examples/diagnostics/async/CosmosDiagnosticsQuickStartAsync.java#L43

Reply:
Thanks, done.